site/gatsby-site/README.md

```diff
@@ -69,3 +72,179 @@ To set up a local development environment for the AIID project, follow these ste
    ```
 
 Follow these steps to get your local environment up and running for development with the AIID project. Make sure to replace the placeholder values in the `.env` file with your actual credentials to ensure proper functionality.
+
+## AIID API 
+
+### Overview
+
+The AIID API is built to facilitate interactions with the AI Incident Database. 
+
+1. **Access the Apollo Explorer**
+
+   Navigate to `http://localhost:8000/graphql` in your web browser. The Apollo Explorer instance should be displayed, allowing you to introspect and run queries against the API.
+
+### Performing Queries
+
+Here is an example query you can run in the Apollo Explorer to retrieve incidents:
+
+```graphql
+query {
+  incident {
+    incident_id
+    title
+  }
+}
+```
+
+### Expected Response
+
+The query should return a response similar to this:
+
+```json
+{
+  "data": {
+    "incident": {
+      "incident_id": 1,
+      "title": "Incident 1"
+    }
+  }
+}
+```
+
+### Project Structure
+
+#### Folders
+
+The api is contained within the `server` directory. The following folders are present in the project:
+
+- **fields/**: Contains the field definitions for the GraphQL root fields.
+- **generated/**: Holds the generated GraphQL types derived from the schema using the GraphQL code generator CLI.
+- **tests/**: Contains the test cases written using Jest.
+- **types/**: Contains the base GraphQL types.
+
+#### Important Files
+
+- `remote.ts`: Contains the remote GraphQL schema.
```

Comment:
Expand what remote means here. These are queries (or "fields") that are forwarded to other API endpoints and this codebase is not directly in control of.

EDIT: Or point the reader to the "stitching" section below that explains perfectly. :-)